simulation/amaru-sim/src/echo/simulate.rs

```diff
@@ -70,33 +69,34 @@ pub fn spawn_echo_node(_node_id: String) -> NodeHandle<EchoMessage> {
 }
 
 /// Generate some input echo messages at different arrival times.
-pub fn echo_generator(rng: &mut StdRng) -> Vec<Reverse<Entry<EchoMessage>>> {
+pub fn echo_generator(rng: &mut StdRng) -> Vec<Entry<EchoMessage>> {
```

Comment:
At this stage I wonder if we should not get rid of the `Echo` stuff

Reply:
This is used to test the simulation framework on a simple system but I also can't tell how useful this is by now.

Reply:
Your call!